stdlib/log-api/src/main/java/org/ballerinalang/stdlib/log/Utils.java

```diff
@@ -27,39 +27,76 @@
  * @since 1.1.0
  */
 public class Utils extends AbstractLogFunction {
+    private static boolean logLevelEnabled = false;
```

Comment:
Why is this maintained at class level? There's no need for this right? Each time a log function is called this is updated, but never used. Plus, this class is meant to be stateless isn't it?

Reply:
Updated in 78c8c82